pkg/services/ngalert/api/tooling/definitions/alertmanager.go

```diff
@@ -504,8 +503,23 @@ func AllReceivers(route *config.Route) (res []string) {
 	return res
 }
 
-type GettableGrafanaReceiver dtos.AlertNotification
-type PostableGrafanaReceiver models.CreateAlertNotificationCommand
+type GettableGrafanaReceiver struct {
+	UID                   string           `json:"uid"`
+	Name                  string           `json:"name"`
+	Type                  string           `json:"type"`
+	DisableResolveMessage bool             `json:"disableResolveMessage"`
+	Settings              *simplejson.Json `json:"settings"`
+	SecureFields          map[string]bool  `json:"secureFields"`
+}
+
+type PostableGrafanaReceiver struct {
+	UID                   string            `json:"uid"`
+	Name                  string            `json:"name"  binding:"Required"`
+	Type                  string            `json:"type"  binding:"Required"`
```

Comment:
I'm not sure if we will still need the `binding` tag after we will use UIDs.

Reply:
Not sure what that tag does, removing it